Recent changes from core (#79)
owncloud/docs-webui: modules/classic_ui/pages/personal_settings/general.adoc

> To change the language for your user account, pick the desired language from the dropdown under Language. 

Reviewer: ```suggestion
To change the language for your user account, pick the desired language from the drop-down menu under Language. 
```

The new language will be auto-saved shortly after you select it.

== Account Related Settings

In the Account section, you will see the following buttons:

image::personal-settings/general/account-related-settings.png[width=450]

* To comply with article 15 of the GDPR framework, ownCloud provides the following buttons to:
+
--
** Request data export
** Request account deletion
--
+
When a user clicks on one of these buttons, the admin group will recieve an email with the users request for further processing.

> * If the user needs to change his role, a field opens post clicking the btn:[Request new role] button, the request can be entered and is sent via email to the admin group for further processing. A possible change can be, that the user needs to be a group admin for a particular custom group.

Reviewer: ```suggestion
* If users need to change their role, they need to click the btn:[Request new role] button, enter the request and send it to the admin group for further processing. A possible change can be that the user needs to be a group admin for a particular custom group.
```
